<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BIDialog</class>
 <widget class="QDialog" name="BIDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>384</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Basic Information Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start this node.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pingButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="readfileButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Read file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shutdownButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Shutdown</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="logView">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>520</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Add new data</string>
   </property>
   <widget class="QLineEdit" name="studyProgram">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="studentId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit2nd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Student id:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2ndItem">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>20</y>
      <width>173</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3rdItem">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Study program:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addDataButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>36</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Add</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>startButton</zorder>
  <zorder>pingButton</zorder>
  <zorder>readfileButton</zorder>
  <zorder>shutdownButton</zorder>
  <zorder>logView</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>startButton</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>readfileButton</tabstop>
  <tabstop>shutdownButton</tabstop>
  <tabstop>studentId</tabstop>
  <tabstop>edit2nd</tabstop>
  <tabstop>studyProgram</tabstop>
  <tabstop>addDataButton</tabstop>
  <tabstop>logView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
